submit-web/src/components/Submission/SubmissionItemTableRow/ProponentSubmissionItemTableRow.tsx

```diff
@@ -124,13 +130,14 @@ export default function ProponentSubmissionItemTableRow({
           </Box>
         </SubmitPrimaryRowTableCell>
         <SubmitPrimaryRowTableCell align="left" width={"10%"}>
-          <Unless
+          <When
             condition={
-              submissionPackage?.submitted_on &&
+              isContactInformation ||
```

Comment:
this logic is too specific. It's open to change and it is coupled with the Management Plan package